landing-page/components/sections/team/TeamSection.tsx

```diff
@@ -115,6 +117,8 @@ const teamData: Record<string, ProfileProps[]> = {
   medical: [
     {
       name: 'Ludwig Maidowski',
+      prefix: 'Dipl.-Jur.',
+      suffix: 'LL.M.',
```

Comment:
'Maître en droit, Paris II'

Reply:
True. I was confused by the differentiation... Resolved 👍🏻